src/app/App.tsx

```diff
@@ -9,27 +9,30 @@ import Ads from './components/Ads/Ads';
 function App(): JSX.Element {
   const [targetMatchId, setTargetMatchId] = useState<string | null>(null);
   return (
-    <div className={classes.container}>
-      <AppHeader className={classes.header} />
-      <main className={classes.main}>
-        {targetMatchId ? (
-          <>
-            <button
-              className={classes.back}
-              onClick={() => setTargetMatchId(null)}
-            >
-              &lt;- Back to feed
-            </button>
-            <MatchDetails matchId={targetMatchId} />
-          </>
-        ) : (
-          <Feed onMatchClick={setTargetMatchId} />
-        )}
-      </main>
-      <aside className={classes.aside}>
-        <User />
-        <Ads />
-      </aside>
+    <div>
```

Comment:
You added a `div` here. I don't think it's required :).

Reply:
You're right :)

Reply:
But didn`t we need the outer div container to arrange everything inside with the grid?

Reply:
@AlexisRoe You're also right. The other div container is still there. I mistakenly inserted a single div tag.